### PR TITLE
Model updates 20210903

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/risk.py [hunk 1]
```diff
@@ -168,7 +175,60 @@ def getModelDefs(self):
...
+                        'doc': 'The number of records compromised which contain PII.',
+                    }),
+                    ('loss:econ', ('econ:price', {}), {
+                        'doc': 'The total economic cost of the compromise.',
+                    }),
+                    ('loss:life', ('int', {}), {
+                        'doc': 'The total loss of life due to the compromise.',
+                    }),
+                    ('loss:bytes', ('int', {}), {
+                        'doc': 'An esimate of the volume of data compromised.',
```
Comment (reign-git) [inline synapse/models/risk.py:215]: change "esimate" to "estimate"

File: synapse/models/risk.py [hunk 2]
```diff
@@ -168,7 +175,60 @@ def getModelDefs(self):
...
+                    ('time', ('time', {}), {
+                        'doc': 'Earliest known evidence of compromise.',
+                    }),
+                    ('duration', ('duration', {}), {
+                        'doc': 'The duration of the compromise.',
+                    }),
+                    ('loss:pii', ('int', {}), {
+                        'doc': 'The number of records compromised which contain PII.',
+                    }),
+                    ('loss:econ', ('econ:price', {}), {
```
Comment (reign-git) [inline synapse/models/risk.py:208]: :loss:* and :ransom:* feel like impacts to me, and therefore candidates for a separate form altogether.

File: synapse/models/risk.py [hunk 3]
```diff
@@ -168,7 +175,60 @@ def getModelDefs(self):
...
+                    ('attacker', ('ps:contact', {}), {
+                        'doc': 'Contact information of the attacker.',
+                    }),
+                    ('campaign', ('ou:campaign', {}), {
+                        'doc': 'The campaign that this compromise is part of.',
+                    }),
+                    ('time', ('time', {}), {
+                        'doc': 'Earliest known evidence of compromise.',
+                    }),
+                    ('duration', ('duration', {}), {
```
Comment (reign-git) [inline synapse/models/risk.py:202]: Curious why we opted for duration vs start and end dates?

File: synapse/models/risk.py [hunk 4]
```diff
@@ -185,23 +248,36 @@ def getModelDefs(self):
...
                     }),
+                    ('compromise', ('risk:compromise', {}), {
+                        'doc': 'A compromise that this attack contributed to.',
+                    }),
                     ('prev', ('risk:attack', {}), {
                         'doc': 'The previous/parent attack in a list or hierarchy.',
                     }),
                     ('actor:org', ('ou:org', {}), {
-                        'doc': 'The org that carried out the attack.',
+                        'deprecated': True,
```
Comment (rakuy0) [inline synapse/models/risk.py:258]: So we have some very nice deprecation docs that would love to have these added to them :smiley:

File: synapse/models/orgs.py [hunk 5]
```diff
@@ -372,7 +372,7 @@ def getModelDefs(self):
                     }),
                     ('type', ('str', {}), {
                         'deprecated': True,
-                        'doc': 'The campaign type.',
+                        'doc': 'Deprecated. Use the :camptype taxonomy.',
```
Comment (vEpiphyte) [inline synapse/models/orgs.py:375]: deprecation doc update

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**